src/styles/main.scss

```diff
@@ -0,0 +1,108 @@
+$day-size: 100px;
+$day-border: 1px;
+$day-gap: $day-border;
+$day-bg: #eee;
+$font-size: 30px;
+$month-length: 31;
+$day-size: 100px;
```

Comment:
The variable `$day-size` is declared multiple times. It's best practice to declare it once to avoid confusion and potential errors. Please remove the redundant declarations.